server/armeria-server/src/main/scala/sttp/tapir/server/armeria/ArmeriaBodyListener.scala

```diff
@@ -0,0 +1,12 @@
+package sttp.tapir.server.armeria
+
+import scala.util.{Success, Try}
+import sttp.monad.MonadError
+import sttp.monad.syntax._
+import sttp.tapir.server.interpreter.BodyListener
+
+private[armeria] final class ArmeriaBodyListener[F[_]: MonadError] extends BodyListener[F, ArmeriaResponseType] {
+  override def onComplete(body: ArmeriaResponseType)(cb: Try[Unit] => F[Unit]): F[ArmeriaResponseType] = {
+    cb(Success(())).map(_ => body)
```

Comment:
In case the response is streaming, is there a way to call the callback once the response is done streaming? In the `Right` case the response is ready so we can call the callback right away